### PR TITLE
Fix checking KMS key based on alias

### PR DESCRIPTION
**Pull request by mikkeloscar**
This fixes the lookup of KMS key alias by just describing the key by name. 

Without this the kms handling would fail in accounts with a lot of aliases because the ones we're looking for from the config may not be returned in the n first results.

### CODE REVIEW

File: sevenseconds/config/kms.py [hunk 1]
```diff
@@ -14,9 +14,9 @@ def configure_kms_keys(account: object, region):
             continue
         key = json.loads(json.dumps(key_config).replace('{account_id}', account.id))
         with ActionOnExit('Searching for key "{}"..'.format(key_alias)) as act:
-            exist_aliases = kms_client.list_aliases()
+            exist_aliases = paginate(kms_client.list_aliases)
             found = False
-            for alias in exist_aliases['Aliases']:
+            for alias in exist_aliases:
```
Comment (aermakov-zalando) [inline sevenseconds/config/kms.py]: `kms.describe_key(KeyId="alias/{}".format(key_alias))` instead? This way you don't need pagination at all, and it's much more efficient.

Comment (aermakov-zalando) [reply sevenseconds/config/kms.py]: Docs [here](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/kms.html#KMS.Client.describe_key).

Comment (mikkeloscar) [reply sevenseconds/config/kms.py]: Good idea. I have changed it!

**Comment by aermakov-zalando** [comment]
👍

**Comment by mikkeloscar** [comment]
:+1:

**Pull request merged**